azurelinuxagent/ga/update.py

```diff
@@ -791,13 +791,14 @@ def __init__(self, path=None, pkg=None, host=None):
 
             msg = u"Agent {0} install failed with exception: {1}".format(
                         self.name, ustr(e))
+            detailed_msg = '{0} {1}'.format(msg, traceback.extract_tb(get_traceback(e)))
```

Comment:
why not adding the detailed message to the local log?

Reply:
it would automatically be added by the `add_event` function as it logs if there was an error

Reply:
then why have the extra warn?

Reply:
I was following the same pattern that we have for try_update_gs function but I guess since the cadence for this is not as frequent I can remove the logger.warn and just keep the log by add_event. Will make the change.